gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTcpChecks.java

```diff
@@ -81,26 +86,26 @@ public void testConnectionCheckerFailures() throws
     for (int i = 0; i < failcount; ++i) {
       // wait for checkinterval, fail till failcount is reached to disable the check
       TimeUnit.MILLISECONDS.sleep(1100);
-      Assert.assertTrue(check.tcpCheck() != ConnectionCheck.TCP_CHECK_SUCCESS);
+      assertNotEquals(ConnectionCheck.TCP_CHECK_SUCCESS, check.tcpCheck());
     }
     // Make the server available and check if the failure contines for disableDuration
     doReturn(mock(Socket.class)).when(check).makeSocket(host, port);
     TimeUnit.SECONDS.sleep(6);
-    Assert.assertTrue(check.tcpCheck() != ConnectionCheck.TCP_CHECK_SUCCESS);
+    assertNotEquals(ConnectionCheck.TCP_CHECK_SUCCESS, check.tcpCheck());
     TimeUnit.SECONDS.sleep(6);
-    Assert.assertTrue(check.tcpCheck() == ConnectionCheck.TCP_CHECK_SUCCESS);
+    assertEquals(ConnectionCheck.TCP_CHECK_SUCCESS, check.tcpCheck());
 
     // We maintain a map of the checkers, so verify that we got the same check back, as before
     Object checkObj = checker.getChecker(host, port, checkInterval,
         failcount, disableDuration);
-    Assert.assertTrue(checkObj == checker.getChecker(host, port, checkInterval,
+    assertEquals(checkObj, checker.getChecker(host, port, checkInterval,
         failcount, disableDuration));
 
-    Assert.assertFalse(checkObj == checker.getChecker(host, port, checkInterval + 10,
+    assertEquals(checkObj, checker.getChecker(host, port, checkInterval + 10,
```

Comment:
@andythsu i believe the test failure was right here. The original was `assertFalse(checkObj == checker...)`, so these should have been changed to `assertNotEquals`. The intent of the test should be clearer this way :)

Reply:
ahh I see. Got tripped over the `assertFalse` and `==`

Reply:
good catch! Thanks @willmostly